collection.php

```diff
@@ -0,0 +1,39 @@
+<?php require('database.php'); ?>
+<html>
+<head>
+    <title>Collection</title>
+    <link rel="stylesheet" type="text/css" href="collection.css">
+</head>
+<body>
+<header>
+    <nav>
+        <img class="guitar" src = "https://w0286994blog.files.wordpress.com/2016/04/scute.jpg?w=676" alt="Guitarlogo"></li>
+        <h1 class="title">THE GUITAR COLLECTION!!!!!</h1>
+    </nav>
+</header>
+<form action="http://localhost:1234/collection/guitarscollection/form.php">
+<button type="submit">the pig wants more rock<br>ADD GUITAR!</button>
+</form>
```

Comment:
Since this is just a link to the form page it can actually just be an A tag. There's no need for this to be a form itself